consumer/pdata/generated_trace.go

```diff
@@ -223,17 +223,17 @@ func (ms ResourceSpans) CopyTo(dest ResourceSpans) {
 type InstrumentationLibrarySpansSlice struct {
 	// orig points to the slice otlptrace.InstrumentationLibrarySpans field contained somewhere else.
 	// We use pointer-to-slice to be able to modify it in functions like Resize.
-	orig *[]*otlptrace.InstrumentationLibrarySpans
+	orig *[]otlptrace.InstrumentationLibrarySpans
```

Comment:
how are you causing changes in this generated file without changes in pdatagen/ ?

Reply:
Manually changed to prove that I can pass tests and see how would it look like. As the description says I am not planning to merge this, just as a way to show how this can be done